src/drive/view/vscodeNotificator.ts

```diff
@@ -9,7 +9,7 @@ export class VSCodeNotificator implements INotificator {
             title: message,
         }, () => {
             const p = new Promise((resolve, reject) => {
-                task.then(() => resolve())
+                task.then(x => resolve(x))
```

Comment:
Is this related to updating some _devDependencies_ on package.json?

Reply:
Yes, I think that is because I updated TypeScript version. VsCode tells that is expected 1 argument but got 0.
But I think that is really correct is return a Promise with void type, i'm going to fix this.